docs/development-setup.md

```diff
@@ -0,0 +1,110 @@
+# Development Setup
+
+This document outlines the steps required to set up your NestJS project for development. By following these steps, you'll be able to run your application locally with the necessary environment variables and database configuration.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following software installed on your machine:
+
+### Node.js and npm
```

Comment:
We can just tell what is required.
Don't have to mention the steps on how to download and all.
They should figure out.
Also for development we suggest using nvm for node.

So better to just tell applications and versions.

Reply:
Updated accordingly